src/App.js

```diff
@@ -1,24 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Login from './components/Login';
+import Register from './components/Register';
+import RequireAuth from './components/RequireAuth';
+import AddCarsForm from './components/cars/AddCarsForm';
+import Session from './components/session/Session';
+import MainPage from './components/cars/MainPage';
+import DeleteCar from './components/cars/DeleteCar';
+import Reservations from './components/reservations/Reservations';
+import CarDetails from './components/details/CarDetails';
+import ReserveNav from './components/reserve/ReserveNav';
+import ReserveDetails from './components/reserve/ReserveDetails';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this main "App" functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪